chromadb/test/property/invariants.py

```diff
@@ -83,7 +83,8 @@ def _field_matches(
     actual_field = result[field_name]
 
     if len(normalized_record_set["ids"]) == 0:
-        assert isinstance(actual_field, list) and len(actual_field) == 0
+        assert isinstance(actual_field, list)
+        assert len(actual_field) == 0
```

Comment:
OK so this assertion really checks whether `actual_field` is an empty list. This pretty much boils down to the single assertion
```suggestion
        assert actual_field == []
```
which is concise and comprehensible. So I think no need to split into two assertions.